### ISSUE TITLE
model: daal4py: Currently broken due to change in 2020.2 release

### ISSUE THREAD

**Issue by johnandersen777** [intel/dffml#801]
https://github.com/IntelPython/daal4py/issues/251

Pinning to `2020.1` as a temporary fix to make the CI pass. We need to fix this before next release. Waiting to hear back on best path forward.

**Comment by mHash1m** [comment]
@pdxjohnny do you suggest I dig into this or should I pick another issue if we want to wait on DAAL people to follow up on this?
